.prettierignore

```diff
@@ -4,3 +4,4 @@ packages/studio-plugin/tests/__fixtures__/PageFile/noReturnParenthesesPage.tsx
 packages/studio-plugin/tests/__fixtures__/PageFile/repeaterPage.tsx
 packages/studio-plugin/tests/__fixtures__/StudioConfigs/malformed/studio.config.js
 packages/studio/src/tailwind-full.css
+.github
```

Comment:
![image](https://github.com/yext/studio/assets/59857107/18c7c335-8e33-4026-ba00-b6d71b9abfa0)

I think that our PAT for the bot does not allow for editing the workflow.  Because of this, I re-added `.github` to prettier.  Should we update the PAT permissions instead?

Reply:
I thought your previous PR had a commit where the GH linting action was able to make a commit that updated the styling for a workflow?

Reply:
I'm slightly confused, because it seems to be able to update some of the work flows (see [this automated commit](https://github.com/yext/studio/pull/325/commits/dcffc7fee3c4ded030ec7d919d5319f8d87230cd)), but not others (see [this failed workflow](https://github.com/yext/studio/actions/runs/5811451007/job/15754715175)). and I'm not completely sure why.

Reply:
It looks like it only causes errors when trying to edit the newly added workflows... looking into it

Reply:
I looked into this with Tom, and it requires editing Slapshot Bot PAT permissions, which might be a little excessive for the ability to lint workflow files.  I think we will keep `.github` in the prettierignore for now.